src/VisualStudio/Core/Test/Progression/CSharpSymbolLabelTests.vb

```diff
@@ -8,9 +8,9 @@ Imports Microsoft.VisualStudio.GraphModel
 Imports Roslyn.Test.Utilities
 
 Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.Progression
-    <[UseExportProvider]>
+    <UseExportProvider, Trait(Traits.Feature, Traits.Features.Progression)>
     Public Class CSharpSymbolLabelTests
-        <Fact, Trait(Traits.Feature, Traits.Features.Progression)>
+        <WpfFact>
```

Comment:
Why are these `WpfFact` now?

Reply:
VisualStudioTestComposition includes JTF. If these are not WpfFacts the threading context is not set as expected:

`System.InvalidOperationException: 'Code cannot switch to the main thread without configuring the IThreadingContext.'`

Reply:
Something changed here. Do you know what it was?

Reply:
📝 Note that the exception you saw isn't caused by the lack of a threading context (or JTF), but rather by the use of `SwitchToMainThreadAsync`. If code uses JTF but never calls `SwitchToMainThreadAsync` it will not have a problem with this threading context configuration.

Reply:
Looks like it's because `VisualStudioTaskSchedulerProvider` is part of the composition now and `QueueTask`, invoked from a workspace event, calls `SwitchToMainThreadAsync`.

Reply:
That's acceptable for this layer, thanks for finding the cause!